Tasks/UseDotNetV2/package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@types/mocha": "^5.2.6",
     "@types/semver": "6.0.0",
+    "@types/uuid": "^3.4.5",
```

Comment:
is this being used by any other task? Do we need to look at OSS governance?

Reply:
Why only @types

Reply:
not using this now.